bazel/cargo/Cargo.toml

```diff
@@ -0,0 +1,68 @@
+[package]
+edition = "2018"
+name = "wasmtime-c-api-bazel"
+version = "0.20.0"
+
+[lib]
+crate-type = ["staticlib", "cdylib"]
+doc = false
+doctest = false
+name = "wasmtime"
+path = "src/lib.rs"
+proc-macro = true
+test = false
+
+[dependencies]
+anyhow = "1.0"
+env_logger = "0.8"
+indexmap = {version = "=1.1.0", features = ["serde-1"]}
+object = {version = "=0.21.1", default-features = false, features = ["write"]}
+once_cell = "1.3"
+wasmtime = {git = "https://github.com/bytecodealliance/wasmtime", tag = "v0.21.0", path = "crates/wasmtime", default-features = false}
+wasmtime-c-api-macros = {git = "https://github.com/bytecodealliance/wasmtime", tag = "v0.21.0", path = "crates/c-api/macros"}
```

Comment:
Aren't those crates available in cargo? Why do we need to download them from git?

Reply:
`wasmtime-c-api-macros` was not published in crateio :disappointed: fixed for `wasmtime` crate